src/script/common/helper.cpp

```diff
@@ -56,7 +56,7 @@ template <> bool LuaHelper::readParam(lua_State *L, int index)
 
 template <> bool LuaHelper::readParam(lua_State *L, int index, const bool &default_value)
 {
-	if (lua_isnil(L, index))
+	if (lua_type(L, index) == LUA_TNONE)
```

Comment:
```suggestion
	if (lua_isnoneornil(L, index))
```

Reply:
The funny thing is, I looked for those functions but I didn't find them. Aaaaaaa
I've updated every `== LUA_TNONE` check